src/components/ForestTypeDescription/utils.js

```diff
@@ -6,6 +6,35 @@ const forestTypeStyles = [styles.rare, styles.often, styles.medium];
 
 export const getStyle = (data, path) => forestTypeStyles[get(data, path, 0)];
 
+export const forestTypeMapping = [
```

Comment:
Isn't this LU specific?

Reply:
We use it in two different locations (/ForestTypeDescription/lu/Tillering, ForestTypeComparison/lu/ForestTypeComparison), so I didn't want to duplicate it

Reply:
Better move it to `lu/utils.js`.